Makefile

```diff
@@ -1,39 +1,70 @@
+# Use /bin/bash instead of /bin/sh
+export SHELL = /bin/bash
+
 ## ========================================
 ## Commands for both workshop and lesson websites.
 
 # Settings
 MAKEFILES=Makefile $(wildcard *.mk)
-JEKYLL=jekyll
-JEKYLL_VERSION=3.7.3
+JEKYLL_VERSION=3.8.5
```

Comment:
I understand from you that GitHub Pages uses v3.9. Should this match with Pages?

Reply:
In fact the latest [`Makefile`](https://github.com/carpentries/styles/blob/gh-pages/Makefile) in the styles repository doesn't specify a specific Jekyll version but now uses a dedicated Carpentries docker image built for the purpose. This `Makefile` should probably be synchronised with that one.

Reply:
Agreed, and good catch.